### PR TITLE
Update PVP notification design.

### PR DESCRIPTION
**Pull request by fotinakis**
Update the design of the PVP notification bar.

## Pictures

screen shot 2018-08-27 at 6 57 13 pm

---

screen shot 2018-08-27 at 6 31 52 pm

---

screen shot 2018-08-27 at 6 31 24 pm

---

screen shot 2018-08-27 at 7 08 27 pm

## Reviewing this PR
_**Is Covered by Automated Tests?: YES/NO** NO

Manually tested.

### CODE REVIEW

File: app/styles/components/_comparison_viewer_full.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .ComparisonViewerFull {
+  min-height: 300px;
```
Comment (fotinakis) [inline app/styles/components/_comparison_viewer_full.scss]: Sneaking in an unrelated bugfix, helps make it much easier to click through fullscreen toggle states on a snapshot with no diffs.

Comment (PixelJanitor) [reply app/styles/components/_comparison_viewer_full.scss]: GET HIM LINDSAY!!! 😉

**Comment by PixelJanitor** [comment]
From a design perspective, I think this looks great. Well done 👍

File: app/styles/components/_comparison_viewer_full.scss (hunk 1 above)
Comment (cadeParade) [reply app/styles/components/_comparison_viewer_full.scss]: I know, but it's only one line @PixelJanitor 😂

Comment (fotinakis) [reply app/styles/components/_comparison_viewer_full.scss]: And I specifically called it out! :)

**Pull request merged**